webapp/src/components/mobilitymap/Map.jsx

```diff
@@ -183,13 +184,14 @@ export default function Map() {
       {error && (
         <CardOverlay>
           <FormError error={error} noMargin component="div" />
-          {readOnlyReason(user, "read_only_zero_balance") && (
-            <div className="text-center mt-2">
-              <Button variant="outline-success" to="/funding" size="sm" as={RLink}>
-                {t("payments:add_funds")}
-              </Button>
-            </div>
-          )}
+          {isUndefined(config.featureMobilityRestricted) &&
```

Comment:
`!config.featureMobilityRestricted`. It may be an empty string if set but empty.